<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steghide-GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="ReloadButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/reload.png</normaloff>:/resources/images/reload.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="AddFilesButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Files</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/add.png</normaloff>:/resources/images/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="EncodeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/encode.png</normaloff>:/resources/images/encode.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="ExtractButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract files</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/extract.png</normaloff>:/resources/images/extract.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="FilesAndFoldersTreeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>121</y>
      <width>431</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="FileTableView">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>101</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>170</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QToolButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/settings.png</normaloff>:/resources/images/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="ConverButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Audio Converter</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/convert.png</normaloff>:/resources/images/convert.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>90</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset resource="imageresources.qrc">
      <normaloff>:/resources/images/help.png</normaloff>:/resources/images/help.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="FreeSpaceProgressBar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Free space for secret files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Carrier files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Secret files:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilesAndFolders_MainPath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageresources.qrc"/>
 </resources>
 <connections/>
</ui>
